Sofascore/Sofascore/Networking/NetworkError.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+enum NetworkError: Error {
```

Comment:
Mozes, ako zelis, istraziti sto je CustomStringConvertible i primjenti to na ovaj enum